code/game/machinery/slide_projector.dm

```diff
@@ -0,0 +1,189 @@
+/obj/item/storage/slide_projector
```

Comment:
Do you need that flag set so you don't beat things with it if you use it from adjacent?

Reply:
Actually works like that already, clicked on adjacent computer and it just projected there.